app/src/js/components/ReconciliationReports/CreateNewReportButton.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import LoadingEllipsis from '../LoadingEllipsis/loading-ellipsis';
+
+const CreateNewReportButton = ({ createReportInflight, getPersistentQueryParams }) => {
```

Comment:
Was this extracted from another component? Or is this new functionality?

Reply:
Ah I see this in list.js. It looks like this is a refined version of what was rendered there previously. The tags have been updated, are there functional/visual changes here?

Reply:
No visual changes just making sure the hooks work properly. Links and buttons for routing work differently because we have to call the React Router and Redux hooks. It's a cleaner way as well to handle React components.